cukinia

```diff
@@ -265,35 +236,32 @@ _colorize()
 	printf "${c}${text}${nc}"
 }
 
-# cukinia_rundir: run scripts in given directory
-# arg: $*: list of directories
-cukinia_run_dirs()
-{
-	local dir
-
-	for dir in "$@"; do
-		_checkdir "$dir" || continue
-		_cukinia_run_dir "$dir"
-	done
-}
-
 # cukinia_run_dir: run all tests file in $1
 _cukinia_run_dir()
 {
 	local dir="$1"
 	local f
 
 	for f in "$dir"/*; do
-		_checkfile "$f" || continue
+		[ -f "$f" ] || continue
 		_cukinia_prepare "External: $f"
 		cukinia_runner "$f"
 	done
 }
 
+# Template generic cukinia_* functions
+for func in $GENERIC_FUNCTS; do
```

Comment:
As much a this is useful. Let try to keep monkey patching at minimum since we would soon end up with unreadable code. But I can see the the usefulness of this one.